documentation-site/examples/layout-grid/custom-css-unit.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+
+import * as React from 'react';
+import {useStyletron} from 'baseui';
+import {Grid, Cell} from 'baseui/layout-grid';
+
+export default () => (
+  <Outer>
+    <Grid
+      gridUnit="em"
+      gridGutters={[1, 2, 2]}
+      gridMargins={[1, 2, 4]}
```

Comment:
I think we should display modifying the theme as the example since that will be the more common use case. The `tsx` version still has that so maybe we can switch back to it.

Reply:
@sandgraham If I use theme version for .js example. Whole page starts using `em` as a grid unit. which cause other custom grid layout to break, Hence I took props approach. I agree to your point of using themes since it would be most common use case. Please help me If I am missing something here.

Reply:
Oh interesting. You tried it with `ThemeProvider` similar to below?
```js
export default () => (
  <ThemeProvider>
    <Outer>
      <Grid>{* ... *}</Grid>
    </Outer>
  </ThemeProvider>
);
```

Reply:
I think this is fine - the other custom example doesn't use theme. I'll make a ticket to look at these again but shouldn't hold this PR up.